examples/console/src/tsconfig.json

```diff
@@ -0,0 +1,24 @@
+{
+  "extends": "../../../tsconfigbase",
+  "compilerOptions": {
+    "outDir": "../build"
+  },
+  "include": ["*"],
```

Comment:
Can these `"include": ["*"],` options be moved to `tsconfigbase`?

Reply:
Sounds good

Reply:
Nope, I tried and the files don't get built.